charts/sonarqube/Chart.yaml

```diff
@@ -22,8 +22,8 @@ annotations:
       url: https://community.sonarsource.com/
   artifacthub.io/changes: |
     - kind: changed
-      description: "replaced hardcoded port's value"
+      description: "updated SonarQube to 9.2.2"
```

Comment:
this is a additive list. please append the previous change, that is not yet released